src/sentry/static/sentry/app/views/onboarding/configure/index.jsx

```diff
@@ -55,29 +42,23 @@ const Configure = createReactClass({
 
     data.org_id = parseInt(organization.id, 10);
     analytics('onboarding.configure_viewed', data);
-  },
 
-  componentWillUpdate(nextProps, nextState) {
-    if (
-      !this.state.isFirstTimePolling &&
-      nextState.hasSentRealEvent == true &&
-      this.state.hasSentRealEvent == false
-    ) {
-      this.redirectUrl();
+    //redirect if platform is not known.
+    if (!params.platform || params.platform === 'other') {
+      this.redirectToNeutralDocs();
     }
+    this.sentRealEvent();
   },
 
-  componentWillUnmount() {
-    clearInterval(this.timer);
-  },
-
-  sentRealEvent(data) {
-    if (data.length == 1) {
-      const firstError = data[0];
-      return !firstError.message.includes('This is an example');
-    } else {
-      return data.length > 1;
+  sentRealEvent() {
+    const project = this.context.organization.projects.find(
+      p => p.slug == this.props.params.projectId
```

Comment:
Is there any possibility that a user that is trialing business (and can use multiple projects feature), enter the onboarding flow with multiple projects?

Reply:
oo good question- they shouldn't be able to afaict since the url that gets them here just grabs the last `project` created from the `organization` object